packages/react-ui/src/app/features/builder/flow-canvas/nodes/step-node.tsx

```diff
@@ -171,6 +122,10 @@ const WorkflowStepNode = React.memo(
       e.stopPropagation();
     };
 
+    const blockSelectorOperation = isAction
+      ? FlowOperationType.UPDATE_ACTION
+      : FlowOperationType.UPDATE_TRIGGER;
```

Comment:
this was previously a ref, and updated from context menu, I don't think we need that behavior since we can just compute the operation here